src/components/skill/SkillCard.astro

```diff
@@ -0,0 +1,30 @@
+---
+interface Props {
+  name: string;
+  description: string;
+  action: { label: string; url: string };
+}
+
+const { name, description, action } = Astro.props;
+---
+
+<div
+  class="overflow-hidden rounded-3xl bg-gray-50 px-6 py-6 text-center shadow-2xl sm:py-8 xl:px-10"
+>
+  <h2
+    class="mx-auto max-w-2xl text-lg font-bold tracking-tight text-gray-800 sm:text-xl md:text-2xl"
+  >
+    {name}
+  </h2>
+  <p class="text-md mx-auto mt-2 max-w-4xl leading-8 text-gray-600">
+    {description}
+  </p>
+  <div class="mt-8">
+    <a
+      href={action.url}
+      class="rounded-md bg-wsi-blue-300 px-8 py-2 text-base font-semibold leading-7 text-white shadow-sm hover:opacity-80 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-white sm:text-lg"
+    >
+      {action.label}
+    </a>
+  </div>
+</div>
```

Comment:
I think the layout looks cleaner with the buttons aligned. You can achieve this by using `flex-col` and `justify-between`:

```suggestion
<div
  class="flex flex-col justify-between gap-8 rounded-3xl bg-gray-50 px-6 py-6 text-center shadow-2xl sm:py-8 xl:px-10"
>
  <div>
    <h2
      class="mx-auto max-w-2xl text-lg font-bold tracking-tight text-gray-800 sm:text-xl md:text-2xl"
    >
      {name}
    </h2>
    <p class="text-md mx-auto mt-2 max-w-4xl leading-8 text-gray-600">
      {description}
    </p>
  </div>
  <a
    href={action.url}
    class="rounded-md bg-wsi-blue-300 px-8 py-2 text-base font-semibold leading-7 text-white shadow-sm hover:opacity-80 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-white sm:text-lg"
  >
    {action.label}
  </a>
</div>
```

Notice, that I also removed `overflow-hidden`. Was there a particular reason for including it?

Reply:
I based this on the existing `Cta` component. Not sure why it's used there